packages/oas-extensions/README.md

```diff
@@ -0,0 +1,15 @@
+# @readme/oas-extensions
+
+A list of extensions that ReadMe has added to extend the OpenAPI Specification.
+
+https://docs.readme.com/docs/openapi-extensions
+
+[![Build](https://github.com/readmeio/oas/workflows/CI/badge.svg)](https://github.com/readmeio/oas/tree/main/packages/oas-extensions)
+
+[![](https://raw.githubusercontent.com/readmeio/.github/main/oss-header.png)](https://readme.io)
```

Comment:
ohhh thank you for this

Reply:
yeah i was sick of seeing that cloudfront.net link

Reply:
I have [an open design request to update this](https://linear.app/readme-io/issue/RM-7811/refreshed-banner-image-for-usage-across-oss-repos) and i was literally thinking of doing this so we can just update the graphic in one fell swoop